test/verify/check-storage-msdos

```diff
@@ -1,92 +0,0 @@
-#!/usr/bin/python3
-# -*- coding: utf-8 -*-
-
-# This file is part of Cockpit.
-#
-# Copyright (C) 2015 Red Hat, Inc.
-#
-# Cockpit is free software; you can redistribute it and/or modify it
-# under the terms of the GNU Lesser General Public License as published by
-# the Free Software Foundation; either version 2.1 of the License, or
-# (at your option) any later version.
-#
-# Cockpit is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public License
-# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
-
-import parent
-from storagelib import *
-from testlib import *
-
-
-class TestStorage(StorageCase):
-
-    def testDosParts(self):
-        m = self.machine
-        b = self.browser
-
-        self.login_and_go("/storage")
-
-        # Add a disk
-        m.add_disk("50M", serial="MYDISK")
-        b.wait_in_text("#drives", "MYDISK")
-        b.click('tr:contains("MYDISK")')
-        b.wait_visible("#storage-detail")
-
-        # Format it with a DOS partition table
-        b.click('button:contains(Create Partition Table)')
-        self.dialog({"type": "dos"})
-        self.content_row_wait_in_col(1, 1, "Free Space")
```

Comment:
Why is this completely removed? I see the point of dropping the "extended partitions" part, but that should go into the corresponding commit, not this one. But creating a DOS partition table etc. still seems valid?

Reply:
Yeah, I was lazy here.  If we keep support for displaying MBR tables, we should create them with `m.execute` and test that they are displayed properly.